pyccel/stdlib/ndarrays/ndarrays.h

```diff
@@ -70,6 +70,12 @@ enum e_types
         nd_cdouble  = 15
 };
 
+enum e_order
+{
+    order_f,
+    order_c,
+};
```

Comment:
These values should be added to the naming module to prevent name collisions

Reply:
Okey!